Survey/Sources/Presentation/Views/AnswerViews/Choice/ChoiceAnswerView.swift

```diff
@@ -10,11 +10,16 @@ import SwiftUI
 
 struct ChoiceAnswerView: View {
 
-    var answerTexts: [String] = []
+    @ObservedObject var input: AnswerViewModel.Input
+    @ObservedObject var output: AnswerViewModel.Output
 
     var body: some View {
         ZStack {
-            PickerView(selections: .constant([1]), data: [answerTexts])
+            if !output.answerTitles.isEmpty {
+                PickerView(
+                    selections: .constant([output.answerTitles.count]),
```

Comment:
I found this logic confusing, why do we pass an array of a single int to `PickerView`? How does the `selections` works underneath (I see no usage yet).

Reply:
Good catch 🙌 . The param PickerView's selection is used to get the selected index and it should be binding a single int instead.
I fixed at https://github.com/nkhanh44/ic-survey-ios/pull/78/commits/192792b789262385009356c80c2daa2daea55b76.